nautilus_trader/adapters/dydx/common/common.py

```diff
@@ -24,6 +24,7 @@ class DYDXOrderTags(NautilusConfig, frozen=True, repr_omit_defaults=True):
 
     is_short_term_order: bool = True
     num_blocks_open: int = 20
+    is_conditional_order: bool = False
```

Comment:
This flag is probably not necessary. Only checking the order type is more robust if the end user forgets to set this property.

Reply:
Yes , I feel so ,this can be simplified using the order type only I am updating it accordingly